dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/AbstractServiceNameMapping.java

```diff
@@ -60,7 +62,13 @@ public abstract class AbstractServiceNameMapping implements ServiceNameMapping,
 
     public AbstractServiceNameMapping(ApplicationModel applicationModel) {
         this.applicationModel = applicationModel;
-        this.mappingCacheManager = new MappingCacheManager("",
+        boolean fileCache = true;
+        Optional<ApplicationConfig> application = applicationModel.getApplicationConfigManager().getApplication();
+        if(application.isPresent()) {
+            fileCache = Boolean.TRUE.equals(application.get().getEnableFileCache()) ? true : false;
```

Comment:
replaced with `Boolean.parseBoolean`

Reply:
application.get().getEnableFileCache() type is Boolean，how to use Boolean.parseBoolean